File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/domain/Orders/Events/OrderEvent.cs
using System;
using ENode.Eventing;

namespace Registration.Orders
{
    public abstract class OrderEvent : DomainEvent<Guid>
    {
        public Guid ConferenceId { get; set; }

        public OrderEvent() { }
        public OrderEvent(Guid conferenceId)
        {
            ConferenceId = conferenceId;
        }
    }
}

File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/domain/Orders/Events/OrderEvent.java
package com.microsoft.conference.registration.domain.Orders.Events;

import org.enodeframework.eventing.DomainEvent;

public abstract class OrderEvent extends DomainEvent<String> {
    public String ConferenceId;

    public OrderEvent() {
    }

    public OrderEvent(String conferenceId) {
        ConferenceId = conferenceId;
    }
}
